src/evaluations/run_evaluation.py

```diff
@@ -80,6 +80,8 @@
                      'The widths of the boxplot in inches.')
 flags.DEFINE_integer('boxplot_size_height_inch', 6,
                      'The widths of the boxplot in inches.')
+flags.DEFINE_bool('frequency_analysis', False,
```

Comment:
I think a flag indicating the type of analysis with a default of "cardinality" would be cleaner.

Reply:
Done.